tests/test_draw.py

```diff
@@ -161,40 +162,138 @@ def assert_call(self):
     plot = Plot()
 
     m = Minuit(cost, 1, 1)
+
+    with patch("IPython.get_ipython") as mock_get_ipython:
+        mock_shell = MagicMock()
+        mock_shell.__class__.__name__ = "ZMQInteractiveShell"
+        mock_shell.config = {"IPKernelApp": True}
+        mock_get_ipython.return_value = mock_shell
```

Comment:
This is nice, can you turn that into a pytest fixture?

Reply:
I did that now and I put it into conftest.py so that both test_draw.py and test_without_ipywidgets.py can use it.